empack_config.yaml

```diff
@@ -0,0 +1,42 @@
+packages:
+  python-dateutil:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*dateutil-zoneinfo.tar.gz'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  matplotlib:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*matplotlibrc'
+      - pattern: '**/matplotlib/mpl-data/**'
+      - pattern: '**/matplotlib/fontlist.json'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  scikit-learn:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '**/sklearn/datasets/**'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  scikit-image:
```

Comment:
Shamelessly adding a new one :) It seems ipympl broke too recently

```suggestion
  ipympl:
    include_patterns:
      - pattern: '**/ipympl/static'
  scikit-image:
```

Reply:
but we still need the *py files,right?

Reply:
Yes, does it only include the matching patterns? It does not use the default ones?

Reply:
the default is just for PKGs that are not explicitly listed. so you still need to add the stuff

Reply:
first, the files are checked against the include-patterns, if any of these matches, the file is considered.
All considered files are checked against all exclude patterns, if none of the exclude pattern are matching, the file will be included

Reply:
Ok :) I updated my suggestion accordingly